sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/Messages.java

```diff
@@ -90,6 +85,11 @@ public enum Messages {
      */
     public static final String EXCEPTION_OCCURRED_WHILE_EMITTING = getMessage("EXCEPTION_OCCURRED_WHILE_EMITTING");
 
+    /**
+     * Encoder is not supported message.
+     */
+    static final String ENCODING_TYPE_NOT_SUPPORTED_ENCODER = getMessage("ENCODING_TYPE_NOT_SUPPORTED_ENCODER");
```

Comment:
This is a breaking change to make a previously `public` constant package-private now.

Reply:
This is not a breaking change as it never shipped. https://github.com/Azure/azure-sdk-for-java/blob/azure-messaging-eventhubs_5.15.0/sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/Messages.java

This Messages class should never have been made part of the public API surface area because it is display messages. The message itself is only used internally.